crates/ty_python_semantic/src/types/infer.rs

```diff
@@ -1734,7 +1736,7 @@ impl<'db, 'ast> TypeInferenceBuilder<'db, 'ast> {
     }
 
     fn infer_region_definition(&mut self, definition: Definition<'db>) {
-        match definition.kind(self.db()) {
+        match &definition.kind(self.db()) {
```

Comment:
Is this spurious? I don't see any other changes to this method or the type being matched on that would require this